app/models/user.rb

```diff
@@ -143,7 +142,7 @@ def ordered_patients(lines = LINES)
     ordered_patients
   end
 
-  def clear_call_list
+  def clean_call_list_between_shifts
```

Comment:
I'm trying to understand the difference between ```#clear_call_list``` and ```#clean_call_list_between_shifts```.

The only time ```#clean_call_list_between_shifts``` is called is during the test cases is because there is additional logic dealing with last sign in?